lib/runner.ex

```diff
@@ -36,17 +37,11 @@ defmodule ElixirBench.Runner do
     {:noreply, client}
   end
 
-  defp process_job(job, client) do
-    %{
-      "id" => id,
-      "repo_slug" => repo_slug,
-      "branch_name" => branch,
-      "commit_sha" => commit,
-      "config" => config
-    } = job
+  defp process_job(job_data, client) do
+    opts = [timeout: @job_timeout, task_fun: &Job.run_job/1]
+    job = build_job_struct(job_data)
 
-    config = Config.from_string_map(config)
-    job = Job.start_job(id, repo_slug, branch, commit, config)
+    job = Job.start_job(job, opts)
```

Comment:
seems like we're missing opportunity to use the great pipe here :) `|>`

Reply:
`revision |> fix()` :heavy_check_mark: